packages/web/src/client/components/Video/VideoPlayer.tsx

```diff
@@ -114,6 +148,44 @@ export const VideoPlayer = ({
         muted
         playsInline
       />
+      <VideoButton
```

Comment:
Maybe this should be using https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls where the ID belongs to the button etc.